src/test/java/juja/sqlcmd/ApplicationTest.java

```diff
@@ -17,10 +17,12 @@
 public class ApplicationTest {
     private static final String LINE_SEPARATOR = System.lineSeparator();
     private static final String DB_CONNECTION_URL = "jdbc:postgresql://127.0.0.1:5432/";
+    private static final String ADMIN_DB_NAME = "postgres";
+    private static final String DB_ADMIN_USER = "postgres";
```

Comment:
May be ADMIN_DB_LOGIN?

Reply:
Sure, it looks much better... or even DB_ADMIN_LOGIN. Thanks!